packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -4245,6 +4245,12 @@ module.exports = function(config, DB) {
           Date.now()
         );
         assert.deepEqual(result, {});
+
+        const updatedAccount = await db.account(account.uid);
```

Comment:
Looks the tests are failing because timestamps are the same. Maybe could use `isAtLeast` or add a tiny delay.